File: P1/E3/ConferenciaBuilder.java
package E3;

class ConferenciaBuilder extends EventoBuilder {
    public void construirNombre(String nombre) {
        evento.setNombre(nombre);
    }

    public void construirFecha(String fecha) {
        evento.setFecha(fecha);
    }

    public void construirUbicacion(String ubicacion) {
        evento.setUbicacion(ubicacion);
    }
}

File: P2/lib/conferencia_builder.dart
import 'evento_builder.dart';

class ConferenciaBuilder extends EventoBuilder {
  @override
  void construirNombre(String nombre) {
    evento.nombre = nombre;
  }

  @override
  void construirFecha(String fecha) {
    evento.fecha = fecha;
  }

  @override
  void construirUbicacion(String ubicacion) {
    evento.ubicacion = ubicacion;
  }
}
